README.md

```diff
@@ -1,98 +1,422 @@
-#### CFPB Open Source Project Template Instructions
+# Macro Polo
 
-1. Create a new project.
-2. Copy these files into the new project.
-3. Update the README, replacing the contents below as prescribed.
-4. Add any libraries, assets, or hard dependencies whose source code will be included
-   in the project's repository to the _Exceptions_ section in the [TERMS](TERMS.md).
-  - If no exceptions are needed, remove that section from TERMS.
-5. If working with an existing code base, answer the questions on the [open source checklist](opensource-checklist.md) 
-6. Delete these instructions and everything up to the _Project Title_ from the README.
-7. Write some great software and tell people about it.
+- Macro Polo
+    - [Requirements](#requirements)
+    - [Installation](#installation)
+    - [Testing Macro Polo](#testing-macro-polo)
+    - [Unit Testing vs Functional Testing](#unit-testing-vs-functional-testing)
```

Comment:
Yeah... that could be "Why Macro Polo? (Unit Testing vs Functional Testing)" or something to that effect.